docs/tutorials/multimodal/object_detection/detection_eval_fasterrcnn_voc.md

```diff
@@ -0,0 +1,108 @@
+# AutoMM Detection - Evaluate Pretrained Faster R-CNN on COCO Format Dataset
```

Comment:
Title and content doesn't match. Title says coco format, but content is voc in voc format

Reply:
Fixed the typo, thanks!